.github/workflows/node.js.yml

```diff
@@ -47,3 +45,21 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - run: npx @adobe/sizewatcher
+  semantic-release:
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: ${{  !contains(github.event.head_commit.message, '[ci skip]') && github.ref == 'refs/heads/master' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+      - name: Use Node.js 12.19
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.19'
```

Comment:
note: newer verisons of semantic release require >node14.17, so this may not work until you update the node version here
https://github.com/semantic-release/semantic-release/blob/HEAD/docs/support/node-version.md

Reply:
Thanks @jdelbick! I ended up using the same (older) version of `semantic-release` as `node-httptransfer` does. That module also uses node `12.19`, and the release process seems to work there.